### PR TITLE
[MRG] Footer update and contact

### PR DESCRIPTION
**Pull request by michelfernandez**
Changed the text for the 2 CTA in Footer.
Only for EN and FR.

### CODE REVIEW

File: site/config.yaml [hunk 1]
```diff
@@ -367,8 +367,8 @@ languages:
           href: /increase-legal-weight/
           target:
       rightSlot:
-        title: Want to schedule a demo session?
-        text: We are pleased to come by to introduce you to Skribble.
+        title: Get in touch?
```
Comment (Kirastina) [inline site/config.yaml:378]:
```suggestion
        title: Want to get in touch?
```

File: site/config.yaml [hunk 2]
```diff
@@ -545,15 +545,15 @@ languages:
...
+        title: Augmenter la force probante?
         text: Identifiez-vous et débloquez le standard de signature électronique le plus élevé.
         button:
           label: S'identifier
           href: /fr/maximiser-force-probante/
           target:
       rightSlot:
-        title: Organiser une démonstration?
-        text: Nous passons volontiers en personne afin de vous présenter Skribble.
+        title: Prendre contact ?
```
Comment (Kirastina) [inline site/config.yaml:567]:
```suggestion
        title: Entrer en contact?
```

File: site/config.yaml [hunk 3]
```diff
@@ -545,15 +545,15 @@ languages:
...
+        text: Nous nous ferons un plaisir de vous contacter pour répondre à vos questions.
         button:
           label: Organiser un appel téléphonique
```
Comment (Kirastina) [inline site/config.yaml:570]:
```suggestion
          label: Organiser un appel
```

File: site/config.yaml [hunk 4]
```diff
@@ -545,15 +545,15 @@ languages:
         href: https://help.skribble.com/meetings/patrick182/consultation-telephone-skribble
     footer:
       leftSlot:
-        title: Maximiser la force probante?
+        title: Augmenter la force probante?
         text: Identifiez-vous et débloquez le standard de signature électronique le plus élevé.
```
Comment (Kirastina) [inline site/config.yaml:561]:
```suggestion
        text: Identifiez-vous et débloquez une norme de signature électronique plus élevée.
```

**Comment by dkumpf1** [comment]
[DEPLOY-PREVIEW]

**Tested: OK**

The footer and the "Contact"-page look good. I tested this on all browsers (Chrome/FF/Chromium/Mobile) and all languages (DE/EN/FR).

**Pull request merged**